SNIClient.py

```diff
@@ -633,7 +633,10 @@ async def game_watcher(ctx: SNIContext) -> None:
         if not ctx.client_handler:
             continue
 
-        rom_validated = await ctx.client_handler.validate_rom(ctx)
+        try:
+            rom_validated = await ctx.client_handler.validate_rom(ctx)
+        except Exception:
+            rom_validated = False
```

Comment:
I think every exception from apworld code should at least get something in the log file. It makes it difficult to debug if there's nothing at all.